terraform/aws/analytical-platform-data-production/create-a-derived-table/iam-policies.tf

```diff
@@ -14,6 +14,10 @@ data "aws_iam_policy_document" "create_a_derived_table" {
       "arn:aws:s3:::mojap-derived-tables",
       "arn:aws:s3:::dbt-query-dump/*",
       "arn:aws:s3:::dbt-query-dump",
+      "arn:aws:s3:::mojap-manage-offences/*",
+      "arn:aws:s3:::mojap-manage-offences",
+      "arn:aws:s3:::mojap-hub-exports/*",
```

Comment:
Hi - Hub exports is a bucket used by a lot of sensitive stuff - this is too broad. Can you rescope to the actual folder used

Reply:
change accordingly